### PR TITLE
bug 1481916 - remove l10n bumper support.

### PR DESCRIPTION
**Pull request by escapewindow**
Non-urgent cleanup; I'm currently pinning both buildbot masters to my puppet env.

### CODE REVIEW

File: modules/l10n_bumper/manifests/services.pp [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-
-    $env_config  = $config::l10n_bumper_env_config[$l10n_bumper_env]
-
-    $owner       = $users::builder::username
-    $basedir     = $l10n_bumper::settings::root
-    $python      = $packages::mozilla::python27::python
-    $mailto      = $l10n_bumper::settings::mailto
-    $config_file = $env_config['config_file']
-    file {
-        '/etc/cron.d/l10n_bumper':
```
Comment (Callek) [inline modules/l10n_bumper/manifests/services.pp]: Note removing this from the puppet manifest will still keep the file around on the host. So might be worth an `ensure=>` directive to remove it (or if you're just going to terminate/disable the instance we probably don't need the extra configuration)

File: modules/toplevel/manifests/mixin/l10n_bumper.pp [hunk 2]
```diff
@@ -1,15 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-class toplevel::mixin::l10n_bumper {
-    include ::l10n_bumper
-
-    ssh::user_privkey {
-        "ffxbld_rsa":
```
Comment (Callek) [inline modules/toplevel/manifests/mixin/l10n_bumper.pp]: same commentary about this key

File: modules/l10n_bumper/manifests/services.pp (hunk 1 above)
Comment (escapewindow) [reply modules/l10n_bumper/manifests/services.pp]: Smart. I was planning on manually disabling and then cleaning up the puppet configs, and terminate the EC2 instances.

File: modules/toplevel/manifests/mixin/l10n_bumper.pp (hunk 2 above)
Comment (escapewindow) [reply modules/toplevel/manifests/mixin/l10n_bumper.pp]: I'll clean this up.

**Pull request merged**